back-office/international/localization/international-localization-localization.md

```diff
@@ -0,0 +1,120 @@
+
+After any modifications applied on local CLDR files, the PrestaShop cache (back-office/advanced-parameters/performance/clear-cache.md) must be cleared to be shown.
+
+**Import**
+
+Pressing the _Import_ button applies the 'Import a localization pack' options. Clicking on it might affect both FrontOffice and BackOffice or change the default price behaviors based on options chose from 'Content to import' and 'Download pack data' fields.
+
+### Configuration
+
+Using this section will give the user the ability to change default localization configuration such as default langiage, country, and currency. Changing this configuration will override the default behaviours.
+
+**Default language**
+
+The _Default language_ droplist displays the languages that are already imported to the shop by user from 'Content to import' section. By default it shows the language that user defined on PrestaShop installation process. Changing it will change the language that is selected by default on FrontOffice (front-office).
+
+**Set language from browser**
+
+The _Set language from browser_ toggle displays a 'YES/NO' toggle button which is set to 'YES' by default. If 'YES' is set, the default language will be changed based on user's default browser langauage automatically on FrontOffice (front-office).
+
+**Default country**
+
+The _Default country_ droplist displays the full list of all countries. The user can select the default country selected on FrontOffice (front-office) from droplist or typing at least one word using the search field on the top of droplist.
+
+**Set default country from browser language**
+
+The _Set default country from browser language_ toggle displays a 'YES/NO' toggle button which is set to 'YES' by default. If 'YES' is set, the default country on FrontOffice (front-office) will be selected automatically based on user's default browser language.
+
+**Default currency**
+
+The _Default currency_ droplist displays the currencies that are already imported to the shop by user from 'Content to import' section. By default, it shows the currency based on default country selected on PrestaShop process. Changing it will change the currency that is selected by default on FrontOffice (front-office).
+
+**Time zone**
+
+The _Time zone_ droplist displays the full list of all standard time zones. The user can select the shop's default Time zone from droplist or typing at least one word using the search field on the top of droplist.
```

Comment:
I wonder if we shouldn't add the GTM/UTC information along with the timezone name.